src/components/ShareStructure/ShareStructure.vue

```diff
@@ -16,8 +16,8 @@
       </div>
 
       <!-- Instructional Text -->
-      <div class="share-info-container info-text pt-6 px-4" v-if="hasRightsOrRestrictions">
```

Comment:
This is no longer conditional?

Reply:
It is not. The verbiage has been updated to be all inclusive.

Reply:
Sounds good.